products/qt.md

```diff
@@ -86,15 +99,22 @@ releases:
 
 ---
 
-> [Qt](https://www.qt.io/) is a widget toolkit for creating graphical user interfaces as well as native cross-platform applications that supports Linux, Windows, macOS, Android and embedded systems. It is available under [both commercial licenses and open-source][license] GPL 2.0, GPL 3.0, and LGPL 3.0 licenses.
+> [Qt](https://www.qt.io/) is a widget toolkit for creating graphical user interfaces as well as
+> native cross-platform applications that supports Linux, Windows, macOS, Android and embedded
+> systems. It is available under [both commercial licenses and open-source GPL 2.0, GPL
```

Comment:
can we revert this ( link part ) ? because it is pushing whole line into next ?
![image](https://user-images.githubusercontent.com/112882/211196003-55747f27-c486-4a3f-a396-5d1b275848db.png)

Reply:
@marcwrobel other than this one LGTM

Reply:
Thank you

Reply:
Sure, done.